src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,447 @@
+| ModeOfAdministration          | REQUIRED              | [string][]    | Mode of administration of the injection (for example, "bolus" or "infusion").                                                                                                                                                                                              |
+| InjectedMassPerWeight         | RECOMMENDED           | [number][]    | Injected mass per kilogram bodyweight.                                                                                                                                                                                                                                     |
+| InjectedMassPerWeightUnit     | RECOMMENDED           | [string][]    | Unit format of the injected mass per kilogram bodyweight (for example, "ug/kg").                                                                                                                                                                                           |
+| SpecificRadioactivityMeasTime | RECOMMENDED           | [string][]    | Time to which specific radioactivity measurement above applies in the default unit "hh:mm:ss".                                                                                                                                                                             |
+| MolarActivity                 | RECOMMENDED           | [number][]    | Molar activity of compound injected. Corresponds to DICOM Tag (0018,1077) Radiopharmaceutical Specific Activity.                                                                                                                                                           |
+| MolarActivityUnit             | RECOMMENDED           | [string][]    | Unit of the specified molar radioactivity (for example, "Bq/g").                                                                                                                                                                                                           |
+| MolarActivityMeasTime         | RECOMMENDED           | [string][]    | Time to which molar radioactivity measurement above applies in the default unit "hh:mm:ss".                                                                                                                                                                                |
+| InfusionSpeed                 | RECOMMENDED           | [number][]    | If given, infusion speed.                                                                                                                                                                                                                                                  |
```

Comment:
Add InfusionStart and InfusionRadioactivity to accommodate bolus-infusion experiments.

```suggestion
| InfusionRadioactivity                 | RECOMMENDED           | [number][]    | Amount of radioactivity infused into the patient. Units should be the same as `InjectedRadioactivityUnit`.                                                                                                                                                                                                                                             |
| InfusionStart                 | RECOMMENDED           | [number][]    | Time of start of infusion with respect to `TimeZero` in the default unit seconds.                                                                                                                                                                                                                                             |
| InfusionSpeed                 | RECOMMENDED           | [number][]    | If given, infusion speed.                                                                                                                                                                                                                                                  |
```

Reply:
I think it's worth being a little bit clearer about this to avoid potential confusion.  Just to be crystal clear that the infusion radioactivity is subsumed by the total injected radioactivity.

Perhaps an extra sentence here: "This value must be less than or equal to the total injected radioactivity (InjectedRadioactivity)."

And even, to be extra clear, an extra sentence under InjectedRadioactivity: "This value includes all injected radioactivity originating from both bolus and infusion."

Thoughts?  I've never actually worked with bolus+infusion data, so I don't know whether this would be obvious or not to people who do.

Reply:
Usually the two measures are reported separately and not as a sum. That is why I originally was eluding to my previous suggestion to disentangle them and put them into an array. But I am a bit worried whether my suggestion will create more confusion regarding  e.g. injectionstart and injectionend, which I guess would also have to be an array in this case. This is currently the solution we have for within-scan challenges (see PharmaceuticalDoseAmount and PharmaceuticalDoseTime). 

But if we agree to keep your suggestion, I agree that the description should be made more clear. I will try to update it. @melanieganz

Reply:
```suggestion
| InfusionRadioactivity                 | RECOMMENDED           | [number][]    | Amount of radioactivity infused into the patient. This value must be less than or equal to the total injected radioactivity (`InjectedRadioactivity`). Units should be the same as `InjectedRadioactivityUnit`.                                                                                                                                                                                                                                             |
| InfusionStart                 | RECOMMENDED           | [number][]    | Time of start of infusion with respect to `TimeZero` in the default unit seconds.                                                                                                                                                                                                                                             |
| InfusionSpeed                 | RECOMMENDED           | [number][]    | If given, infusion speed.                                                                                                                                                                                                                                                  |
```